### DISCUSSION TITLE
Ganach CLI on WSL

### DISCUSSION THREAD

**Discussion by sl00k** [smartcontractkit/full-blockchain-solidity-course-py#381] [Q&A] [answered]
Hello,

I get the following error. It worked while I was working on Windows only, but after setting up WSL I get the error from Ganache

![image](https://user-images.githubusercontent.com/33859801/142184267-b753feca-d51d-48a8-b52f-8ad90fd5ed36.png)

**Comment by sl00k** [comment, marked as answer]
running this command fixed it 
export NODE_OPTIONS=--openssl-legacy-provider

**Reply by sl00k** [reply]
but i have to post it everytime :(

**Reply by cromewar** [reply]
you just have to add that command to ~/.bashrc if you are using linux or ~/.zshrc if you are on MAC.

You must open that file using nano, or vim deppending what you have for example:

`nano ~/.bashrc`

and add that command and the end of the file, save and restart the console.
